api-specificatie/openapi.yaml

```diff
@@ -1640,10 +1603,6 @@ components:
           type: array
```

Comment:
Adresseerbareobjecten zouden we hier toch niet meer doen, omdat dit er nogal veel kunnen zijn? De aan een pand gerelateerde adresseerbare objecten kunnen met het pand ID worden opgevraagd via de /adresseerbareobjecten endpoint, zodat paginering toegepast kan worden, dat kan hier niet.

Hetzelfde geldt voor adressen overigens, die kunnen via /adressen worden opgevraagd met paginering.